appinfo/info.xml

```diff
@@ -15,10 +15,12 @@
         <prelogin/>
     </types>
     <category>security</category>
-    <screenshot>https://raw.githubusercontent.com/owncloud/security/43c55325cf09e5a8b81fafc358b08701aba81173/screenshots/settings.png</screenshot>

```

Comment:
there can be multiple screenshots - keep it

Reply:
but as second entry - the first one is used as the main image on the marketplace